pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+
+## Instructions and Submission Templates
+Three types of predictions will be accepted. Participants are encouraged to submit their results in all or multiple submission types as it fits to their prediction methods.
+
+#### Type I - microscopic pKas and related microstates
+Predicting microscopic pKa values and related microstates structures. Different protonation states and tautomers constitute different microstates. 
+- Fill the `typeI_microscopic_pKas_and_microstates.csv` template for all molecules.
+- For each molecule, report as many microscopic pKas based on your best guest.
+- Record the pair of microstates IDs of microstate structures pairs (protonated HA and deprotonated A) associated for each microscopic pKa. To determine the microstate ID for your predicted structure check the spreadsheets in [SAMPL6/physical_properties/pKa/microstates](SAMPL6/physical_properties/pKa/microstates).
+- If your predicted structure is not included in the list, contact us to [make a request for new microstate](mehtap.isik@choderalab.org). See more details are in missing microstates section.
+- Report microscopic pKa values with 2 decimals (e.g. 10.71).
+- Reporting standard error of the mean (SEM) is optional and should be reported with 2 decimals. (e.g. 1.02).
+
+#### Type II - microstate populations as a function of pH
+Predicting fractional population of microstates between pH interval 2 to 12 with 0.1 pH increments, based on microscopic pKa predictions.
+- Fill the `typeII_microstate_fractional_populations.csv` template file for all molecules and microstates you have predictions for.
+- To determine the microstate ID for your predicted microstate populations check the spreadsheets in [SAMPL6/physical_properties/pKa/microstates](SAMPL6/physical_properties/pKa/microstates).
+- If your predicted structure is not included in the list, contact us to [make a request for new microstate](mehtap.isik@choderalab.org). See more details are in missing microstates section.
+- Report natural logaritm of fractional microstate populations with scientific notation and 3 decimals of precision, for each pH.  
+e.g. For a molecule with only two possible microstates A and B `ln(fractional microstate population) = ln(N_A/(N_A+N_B))` where `N_A` and `N_B` represent number of molecules at microstate A and B.   
+At a pH where 90.0% of the molecules are in microstate B and 10.0% of molecules are in state A  `ln(fractional microstate A population) = ln(0.100/(0.100+0.900)) = -2.30E+00`.  
+- If your estimate of `fractional microstate population` is 0, thus `ln(fractional microstate population) = ln(0)`, report as `-infinity`.
+- Do not report SEM in this submission type.
```

Comment:
what is SEM? Maybe I missed something above.

Reply:
Oh, I see that you define this as standard error of the mean. I think it would be more clear to just write that out. 
Replace with something like: "Standard error will not be reported for this submission type."

Why aren't you asking for an error estimate on this type?

Reply:
There were too many columns to report already for pH interval 2-12 with 0.1 increments. If we ask for standard error we have to ask for estimation of standard error for every pH point for every microstate. I thought we don't have good way to use that information to justify asking for so much.